src/install_packages.rs

```diff
@@ -102,6 +110,12 @@ pub(crate) async fn install_packages(
                     url = style::url(build_download_url(package_to_install))
                 ));
             }
+            for download_url in &packages_to_download {
+                print::sub_bullet(format!(
+                    "Package from {url}",
+                    url = style::url(download_url.to_string())
+                ));
```

Comment:
Suggestion: Warn when using an `http` instead of an `https` URL.

Reply:
Question: the `http` protocol is necessary for the repositories for mirroring, etc. and secure since there is a chain of trust and verification built-in for repository packages. For download urls though, should we just limit support to `https`? We could always add `http` back in later.